src/cue.c

```diff
@@ -22,68 +22,31 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <libcue.h>
 
 gboolean
 cue_read_file(const char *cue_filename, TrackBreakList *list)
 {
     FILE *fp;
-    /* These buffers must have the same length */
-    char line_buf[1024];
-    char buf[1024];
-    int next_track = 1;
 
     fp = fopen(cue_filename, "r");
-    if (!fp) {
-        return FALSE;
-    }
-
-    track_break_list_clear(list);
-
-    /* Read the first line: FILE "foo.wav" WAVE */
-    if (!fgets(line_buf, sizeof(line_buf), fp)) {
-        fclose(fp);
-        return FALSE;
-    }
-
-    /* Check that it starts with FILE and ends with WAVE */
-    if (sscanf(line_buf, "FILE %s WAVE", buf) != 1) {
-        goto error;
-    }
+    if (!fp) return FALSE;
 
-    while (!feof(fp)) {
-        int N;
-        int read;
-        guint offset;
+    Cd *cd = cue_parse_file(fp);
```

Comment:
Can `cue_parse_file` return NULL on error? Do we need to handle that (cleanup + return `FALSE`)?

Reply:
Good catch! Yeah, this can return NULL on error.